src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
tests file name should have ending .spec.js

Reply:
Changed